ui/src/style/drawer.less

```diff
@@ -26,3 +23,9 @@
 .drawer__content {
   padding: 0 @padding;
 }
+
+.drawer__close_icon {
+  position: fixed;
+  border-radius: 100px;
```

Comment:
Don't icon buttons already have rounded corners?

Reply:
Yes it appears there is a border-radius: 50% on the root of all MUI buttons. For some reason it is not actually working. I will investigate :/